app/gui/src/dashboard/layouts/Settings/ProfilePictureInput.tsx

```diff
@@ -26,30 +28,41 @@ export default function ProfilePictureInput(props: ProfilePictureInputProps) {
 
   const uploadUserPicture = useMutation(backendMutationOptions(backend, 'uploadUserPicture'))
 
-  const form = Form.useForm({
```

Comment:
what's the reasoning behind inlining the form?

Reply:
We don't need to have a form in here. Inline form is the default behavior, it has better performance.